tvb_library/tvb/datasets/zenodo.py

```diff
@@ -0,0 +1,111 @@
+# code from https://github.com/space-physics/pyzenodo3 and https://github.com/space-physics/pyzenodo3/pull/9
+
+
+import requests
+import re
+import pooch
+from pathlib import Path
+import json
+
+BASE_URL = "https://zenodo.org/api/"
+
+
+class Record:
+    def __init__(self, data, base_url: str = BASE_URL) -> None:
+        self.base_url = base_url
+        self.data = data
+        self.file_loc = {}
+
+    def describe(self):
+        return self.data['metadata']['description']
+
+
+    def __str__(self):
+        return json.dumps(self.data, indent=2) 
+
+    def download(self):
+
+        if 'files' not in self.data:
+            raise AttributeError("No files to download! Please check if the record id entered is correct! or the data is publically accessible")
+
+        
+
+        for file in self.data["files"]:
+            url = file['links']['self']
+            known_hash = file['checksum']
+            file_name = file['key']
+            
+            file_path = pooch.retrieve(url= url, known_hash= known_hash, progressbar=True)
+
```

Comment:
Due to this, the files are downloaded to `~/.cache/pooch`, I plan to change this to `~/.cache/tvb`.